mappings/net/minecraft/entity/Entity.mapping

```diff
@@ -193,6 +195,10 @@ CLASS net/minecraft/class_57 net/minecraft/entity/Entity
 	METHOD method_1358 bypassesSteppingEffects ()Z
 	METHOD method_1359 isOnFire ()Z
 	METHOD method_1360 hasVehicle ()Z
+	METHOD method_1361 setEquipmentStack (III)V
+		ARG 1 armorSlot
```

Comment:
pretty sure this isn't armorSlot but rather equipmentIndex where 0 is your hand... perhaps.

Reply:
NAH this is right, but 0 is your hand lol